src/toolbar/ToolBar.js

```diff
@@ -119,7 +119,9 @@ class ToolBar extends React.Component{
             <i className="glyphicon glyphicon-trash"></i> Delete
           </button>:null;
     var searchTextInput = this.props.enableSearch?
-      <input type='text' placeholder={this.props.searchPlaceholder?this.props.searchPlaceholder:'Search'} onKeyUp={this.handleKeyUp.bind(this)}/>:null;
+      <div className="form-group form-group-sm">
+        <input className="form-control" type='text' placeholder={this.props.searchPlaceholder?this.props.searchPlaceholder:'Search'} onKeyUp={this.handleKeyUp.bind(this)}/>
```

Comment:
@imaireaux I understand your expectation but I'm afraid it'd bloat the props if we passed this class name to the component. The more general solution to your problem would be to add a class to the main container and write a CSS selector to affect search input specifically. Something like this:

```
.react-bs-container.my-table .tool-bar .form-group input {
  color: red;
}
```